mailhandler/spameater

```diff
@@ -33,12 +33,16 @@ my $configfile = "/path/to/spamgourmet.config";
 
 ##################### try not to edit anything below ##############
 
-use vars qw {$config $util $mailer $checker};
+use vars qw {$config $util $mailer $checker $extradebug $debugstderr};
 $config = Mail::Spamgourmet::Config->new(configfile=>$configfile,mode=>0);
 
-my $extradebug = 0;
+$extradebug //= 0;
+$debugstderr //= 0;
+
+$config->debugstderr($debugstderr);
 $config->debug('spameater started') if $extradebug;
 
+
```

Comment:
Agree that the line 34 comment is not very useful.

Reply:
The comment is aimed at system administrators who aren't developing the system. They nonetheless need to edit the source code so that the configuration file can be found but they having done that they should leave the rest of the code alone. Ideally we'd improve the way the whole installation was handled e.g. using a proper package, but improving the comment might also be wortwhile.

Reply:
I'm changing this to say "configuration section to be edited locally" / "end of configuration section.